tests/unit_tests/conftest.py

```diff
@@ -163,3 +164,38 @@ def dummy_query_object(request, app_context):
         _datasource_dao=unittest.mock.Mock(),
         session_maker=unittest.mock.Mock(),
     ).create(parent_result_type=result_type, **query_object)
+
+
+def with_feature_flags(**mock_feature_flags):
+    """
+    Use this decorator to mock feature flags in tests.integration_tests.
+
+    Usage:
+
+        class TestYourFeature(SupersetTestCase):
+
+            @with_feature_flags(YOUR_FEATURE=True)
+            def test_your_feature_enabled(self):
+                self.assertEqual(is_feature_enabled("YOUR_FEATURE"), True)
+
+            @with_feature_flags(YOUR_FEATURE=False)
+            def test_your_feature_disabled(self):
+                self.assertEqual(is_feature_enabled("YOUR_FEATURE"), False)
+    """
+
```

Comment:
The decorator just mocked a single FF item, in order to support more than 1 FF item, I suggest to provide a DICT structure as a parameter.

Reply:
Lets tackle this in a follow-up PR given that this is a copy/paste of the decorator found in the integration tests.